source/release-notes/1.7.txt

```diff
@@ -0,0 +1,49 @@
+.. _c2c-release-notes-1.7:
+
+===================================
+Release Notes for mongosync 1.7
+===================================
+
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. _1.7.0-c2c-release-notes:
+
+1.7.0 Release
+-------------
+
+**October 18, 2023**
+
+
+Other Notes
```

Comment:
I wonder if this heading is necessary?

Reply:
It is to ensure the content jives with Jason's RN additions (which I just pulled in from master), since this stuff is meant to occur after the headline features.